docs/api/commands/writefile.mdx

```diff
@@ -213,8 +213,7 @@ cy.readFile(filename, null).then((obj) => {
 
 ### Assertions [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Assertions)
 
-- `cy.writeFile()` will only run assertions you have chained once, and will not
-  [retry](/guides/core-concepts/retry-ability).
+- `cy.writeFile()` has no implicit assertions.
```

Comment:
writeFile yields `null`, and makes no implicit assertions about the subject it yields.

In my mind, and implicit assertion isn't "any way a command could fail" - it's "things Cypress already checks about the subject". You never need to write ".should('exist')" about subjects, because Cypress implicitly asserts that they exist. In this case the subject is `null`, and there are no implicit assertions that you could make explicit about it.